docs/core/docker/publish-as-container.md

```diff
@@ -115,7 +136,21 @@ The preceding .NET CLI command publishes the app as a container:
 > [!TIP]
 > Depending on the type of app you're containerizing, the command-line switches (options) might vary. For example, the `/t:PublishContainer` argument is only required for non-web .NET apps, such as `console` and `worker` templates. For web templates, replace the `/t:PublishContainer` argument with `-p:PublishProfile=DefaultContainer`. For more information, see [.NET SDK container builds, issue #141](https://github.com/dotnet/sdk-container-builds/issues/141).
 
-The command produces output similar to the following:
+The command produces output similar to the example output:
+
+:::zone pivot="dotnet-8-0"
+
+```dotnetcli
+Determining projects to restore...
+  All projects are up-to-date for restore.
+  DotNet.ContainerImage -> .\Worker\bin\Release\net8.0\linux-x64\DotNet.ContainerImage.dll
+  DotNet.ContainerImage -> .\Worker\bin\Release\net8.0\linux-x64\publish\
+  Building image 'dotnet-worker-image' with tags 1.0.0 on top of base image mcr.microsoft.com/dotnet/aspnet:8.0
+  Pushed container 'dotnet-worker-image:1.0.0' to Docker daemon
```

Comment:
nit: in .NET 8 the default tag is `latest`, not the application version. We changed this so that dev-inner-loop experiences would have a more Docker-like feel - `docker build . -f Dockerfile` builds an image with the `latest` tag as well.